language/diem-tools/oncall-trainer/src/experiments/mod.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use cli::client_proxy::ClientProxy;
+use gag::Gag;
+use rustyline::{config::CompletionType, Config, Editor};
+use std::{collections::HashMap, sync::Arc};
+
+mod get_sequence_number;
+mod get_writeset_version;
+
+pub trait Experiment {
+    fn name(&self) -> &'static str;
+    fn description(&self) -> &'static str;
```

Comment:
What would you think of having these as trait consts instead of functions here?

Reply:
I think I would keep the function form. I've been a bit torn of whether to use `&'static str` vs `const` for a while and I think function is a bit more flexible in the long run. For a more complicated test the description might be dependent on the test body itself, which is the extra flexibility function will buy us potentially in the future.